plugins/pencilblue/controllers/preview.js

```diff
@@ -56,10 +55,6 @@ module.exports = function PreviewModule(pb) {
         });
     };
 
-    Preview.prototype.getPageTitle = function() {
-        return article.name;
-    };
```

Comment:
The case was redundant,  as it uses the same as the default case, so why have it? (and bithound didn't like the syntax for whatever reason) And if I'm not mistaken this function isn't